patches/@openeth+truffle-typings+0.0.6.patch

```diff
@@ -1,23 +0,0 @@
-diff --git a/node_modules/@openeth/truffle-typings/index.d.ts b/node_modules/@openeth/truffle-typings/index.d.ts
```

Comment:
I'm not sure we can remove this patch otherwise, we won't be able to change the openeth/truffle-typings. Maybe we could do this in a new package but eventually, this should be handled in a different task if this was the idea